spec/features/circle_check_score_update_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe 'updating a circle check score' do 
+
+  before :each do 
+    create :circle_check_score
+  end
+
+
+  context 'with admin privilege' do 
+    it 'updates circle check score' do 
+      when_current_user_is :admin
+      visit circle_check_scores_url
+      fill_in 'circle_check_score_defects_found', with: '4'
+      click_on 'Save score'
+      expect(page).to have_text 'Score was saved.'
+      input = find_field :circle_check_score_defects_found
+      expect(input.value).to eql '4'
```

Comment:
Yes, that's what I'd recommend.
Am 24.05.2016 12:16 schrieb "AkivaGreen" <notifications@github.com>:

> In spec/features/circle_check_score_update_spec.rb
> <https://github.com/umts/umasstransit.rodeo/pull/33#discussion_r64424003>:
>
> > +describe 'updating a circle check score' do
> > +
> > +  before :each do
> > +    create :circle_check_score
> > +  end
> > +
> > +
> > +  context 'with admin privilege' do
> > +    it 'updates circle check score' do
> > +      when_current_user_is :admin
> > +      visit circle_check_scores_url
> > +      fill_in 'circle_check_score_defects_found', with: '4'
> > +      click_on 'Save score'
> > +      expect(page).to have_text 'Score was saved.'
> > +      input = find_field :circle_check_score_defects_found
> > +      expect(input.value).to eql '4'
>
> Could it work, if I create it with a value that would be different from
> the 4? Like per say 3? That way it would know if it updates or not.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/umts/umasstransit.rodeo/pull/33/files/ffa49c108a58a9721502158cea2e40a281803aad#r64424003>
>